deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/SpecItemConfig.java

```diff
@@ -59,6 +59,15 @@ public class SpecItemConfig {
     @ConfigItem(name = "return-response")
     public Optional<Boolean> returnResponse;
 
+    /**
+     * ClientHeaderFactory class that should be used in @RegisterClientHeaders.
+     * Full class name is expected
+     * By default generated AuthenticationPropagationHeadersFactory class is set.
+     * Can also be set to 'none' to use default microprofile implementation (DefaultClientHeadersFactoryImpl)
+     */
```

Comment:
This is confusing, IMO.

So I can add my own, we can have a default, and another "default", which is `none`?

Reply:
Maybe replace `none` with `microprofile-default`, or just `microprofile`?

Reply:
Sorry for confusing doc, is this one better?
"If option is not set then AuthenticationPropagationHeadersFactory is generated and used as value for @RegisterClientHeaders
If set to 'none' then @RegisterClientHeaders will be generated without value and default microprofile implementation will be used"

Reply:
what about here @hbelmiro is new text ok or should i change 'none'?

Reply:
The new text seems fine to me. WDYT @ricardozanini?